plasma_framework/contracts/src/framework/registries/ExitGameRegistry.sol

```diff
@@ -27,6 +27,15 @@ contract ExitGameRegistry is Operated {
         _;
     }
 
+    /**
+     * @notice Exposes information about exit games quarantine
+     * @param _contract address of exit game contract
+     * @return A boolean value denoting whether contract is safe to use, is not under quarantine
+     */
+     function isSafeToUse(address _contract) public view returns (bool) {
```

Comment:
I would name this `isExitGameSafeToUse` to avoid possible name collision with any other mixins we combined to `PlasmaFramework`.

Another minor comment is that this function might be able to use `external` instead of `public`. But I just realized all functions in this file defaults to public instead of external (sigh, my bad) so I will leave this to you. I don't think it matters that much here.

Reply:
I'm leaving as is until I gain my own opinion on this :) Thanks for pointing me this concern!